test/index.js

```diff
@@ -1006,3 +1006,54 @@ test('remark-validate-links', async (t) => {
 
   t.end()
 })
+
+test('links to other branches', async (t) => {
+  const {stderr} = await exec(
+    [
+      bin,
+      '--no-config',
+      '--no-ignore',
+      '--use',
+      '"../../index.js=repository:\\"wooorm/test#main\\""',
+      '--use',
+      '../sort.js',
+      'github.md',
+      'examples/github.md'
+    ].join(' ')
+  )
+  t.deepEqual(
+    strip(stderr),
+    [
```

Comment:
this is a rather verbose test. What’s okay, and should (not) be okay? What’s broken and should (not) be broken?